VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegment.java

```diff
@@ -111,6 +111,8 @@ Properties trackVideoOrientation(String videoId, Double currentTime,
 
     Properties courseShared(String courseId, String socialNetwork);
 
+    Properties courseDetailShared(String courseId, String category, String about_url, String type);
```

Comment:
IMO, we should probably get rid of the social sharing stuff. Some of it might come back around when we do teams, but it was written a long time ago by an outside team and largely depends on a facebook API that no longer exists. I'd rather we just dredged up any code from git if we really need it.